charts/microservice/values.yaml

```diff
@@ -112,12 +112,29 @@ deployment:
 
   tolerations: []
 
-  affinity: {}
+  affinity: 
```

Comment:
Let's make this setting as default in the deployment helm template. If someone wants to override the default setting, he can use these parameters to override.

Reply:
I have done the changes 
To make the affinity and tolerations 
in deployment.yaml
 affinity:
        {{- toYaml (merge .Values.deployment.affinity (default dict .Values.deployment.affinity)) | nindent 8 }}
      tolerations:
        {{- toYaml (merge .Values.deployment.tolerations (default dict .Values.deployment.tolerations)) | nindent 8 }}
        
affinity and tolerations settings have default values defined in values.yaml, and these settings can be overridden by user-specified values. The merge function combines the defaults with the user-provided values, ensuring that the user can override the default settings if needed.